src/main/java/mate/academy/spring/model/User.java

```diff
@@ -42,18 +44,12 @@ public void setPassword(String password) {
         this.password = password;
     }
 
-    public String getSalt() {
-        return salt;
-    }
-
-    public void setSalt(String salt) {
-        this.salt = salt;
-    }
-
     @Override
     public String toString() {
         return "User{"
                 + "id=" + id
-                + ", email='" + email + '\'' + '}';
+                + ", email='" + email + '\''
+                + ", password='" + password + '\''
```

Comment:
Consider removing the 'password' field from the 'toString' method to avoid exposing sensitive data